lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -476,6 +493,22 @@
       username: user
       password: password
 
+# create vm from template and create all disks on specific storage domain
+ovirt_vms:
+  name: vm_test
+  cluster: mycluster
+  template: mytemplate
+  operating_system: other_linux
+  type: server
+  cpu_cores: 1
+  cpu_sockets: 1
+  state: stopped
+  clone: True
+  timeout: 600
```

Comment:
Maybe I would remove the parameters which are not relevant to the example, to make it more clear?